README.md

```diff
@@ -10,6 +10,15 @@ You can download the module from Github by cloning it directly or saving as ZIP
 ### Copy Files
 To install the module, please copy content of the `src` directory into the main directory of Magento 1 shop.
 
+### Installing dependencies
+Since this plugin using `Fact-Finder php-communication-sdk` package as a dependency it should be installed manually.
+Because of Magento 1 does no support compose autoloader You need to reproduce follow steps:
+- login to the server where Magento 1 is installed via SSH
+- copy `composer.json` file to the root of Your Magento 1 directory
```

Comment:
This will really piss of users. Maybe better instead of copying compoer.json somewhere and running composer install just use composer require?
```suggestion
- run composer require: "omikron/factfinder-communication-sdk"
```